subset/alliedSwitch/test_pswitch

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
```

Comment:
also should name this (and others), something like test_switch_at -- since there will be other pswitches!

Reply:
changed test to switchtest, with a view to adding in cisco in the near future
complete.

Also to add when my test_xxxxxxxx is over a certain amount of characters it doesnt build ?

Reply:
If there is an issue there with test-name size that's blocking you, please file a GitHub issue. It should be handled/resolved separately from this PR!